docs/sources/reference/components/prometheus.remote_write.md

```diff
@@ -472,6 +474,53 @@ If the component falls behind more than one third of the data written since the
 last truncate interval, it is possible for the truncate loop to checkpoint data
 before being pushed to the remote_write endpoint.
 
+### Tuning `max_shards`
```

Comment:
We may want to also link to official Prometheus tuning docs: https://prometheus.io/docs/practices/remote_write/ 

But I find your explanation a bit more practical.

Reply:
I read it, but I'm not sure it's worth linking it, at least with regards to `max_shards`. I don't think it's saying much about it.

Reply:
Not feeling too strong about it, so let's not block the PR on this. 

In general this page could have some 'further reading' section though with links, perhaps people may find their answers in the upstream that may be more up-to-date in practice. @clayton-cornell do we have any thoughts about it?

Reply:
If the external info doesn't really add a lot, then we don't really need to link to it. 

It's easy to add: `Refer to the Prometheus [remote write tuning](https://prometheus.io/docs/practices/remote_write/) documentation for more information.`  at the bottom of this section if it helps the users.